middleware/proxy/upstream.go

```diff
@@ -175,6 +181,12 @@ func parseBlock(c *parse.Dispenser, u *staticUpstream) error {
 			return c.ArgErr()
 		}
 		u.IgnoredSubPaths = ignoredPaths
+	case "proxy_redirect":
```

Comment:
Those seem like a limitation in the web app, like trying to work around something that the backend is enforcing.